internal/services/shared/errors.go

```diff
@@ -176,8 +176,6 @@ func RewriteError(ctx context.Context, err error, config *ConfigForErrors) error
 	case errors.As(err, &datastore.ErrCounterNotRegistered{}):
 		return spiceerrors.WithCodeAndReason(err, codes.FailedPrecondition, v1.ErrorReason_ERROR_REASON_COUNTER_NOT_REGISTERED)
 
-	case errors.As(err, &graph.ErrInvalidArgument{}):
```

Comment:
Why removing the case, isn't it being potentially used elsewhere upstream?

Reply:
I removed the error type